### PR TITLE
Generate data from command line

### PR DESCRIPTION
**Pull request by fedetask**
`generate_data.py` can be fully used with command line arguments. See the `README.txt` for a description.

### CODE REVIEW

File: generate_data.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+        env_params = _read_params(DEFAULT_ENVIRONMENTS_PATH + args.env[0] + '.yaml')
+        params['environment'] = env_params['environment']
+    if args.env_spec is not None:
+        for spec in args.env_spec:
+            key, value = spec.split(':')
+            params['environment'][key] = ast.literal_eval(value)
+
+
+if __name__ == '__main__':
+    DATASETS_ROOT = os.path.join(os.path.dirname(__file__), 'datasets')
```
Comment (charlio23) [inline generate_data.py:74]: Could we make this be a parameter in the yaml file?

In my case i need to load the datasets in a different location than "datasets". This change also applies to offline dataset loader called in the train.py file. I might open a new PR for that.

File: experiment_params/default_environments/three_bodies.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+environment:
+  name: "NObjectGravity"
+  mass: [0.5, 0.5, 0.5]
```
Comment (charlio23) [inline experiment_params/default_environments/three_bodies.yaml]:
```suggestion
  mass: [1., 1., 1.]
```

File: experiment_params/default_environments/two_bodies.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+environment:
+  name: "NObjectGravity"
+  mass: [0.5, 0.5]
```
Comment (charlio23) [inline experiment_params/default_environments/two_bodies.yaml]:
```suggestion
  mass: [1., 1.]
```

**Review by charlio23 — COMMENTED** [review]
Nice implementation!

File: experiment_params/default_environments/three_bodies.yaml (hunk 2 above)
Comment (fedetask) [reply experiment_params/default_environments/three_bodies.yaml]: Thanks, I was not sure about the default parameters. Are all the others okay?

File: generate_data.py (hunk 1 above)
Comment (fedetask) [reply generate_data.py:74]: Can I make it a command line argument? Because it would be weird to have it the `default_online.yaml`

Comment (charlio23) [reply generate_data.py:74]: oh that would be perfect.

**Review by OleguerCanal — COMMENTED** [review]
generate_data.py is missing docstrings but I think for now its ok

**Review by charlio23 — APPROVED** [review]
Nice!

**Pull request merged**